packages/functions/src/resources/bot/actions/sendToChannel.ts

```diff
@@ -0,0 +1,47 @@
+import {createMessage} from "../dataModifyers/createMessage";
+import {formatLanguageSpecificData}
+  from "../dataModifyers/formatLanguageSpecificData";
+import * as admin from "firebase-admin";
+import axios from "axios";
+
+export const sendToChannel =
+ async (eventData: admin.firestore.DocumentData) => {
+   if (!eventData || eventData.telegramMessageId) {
+     return;
+   } else {
+     const event = eventData;
+     const formattedLanguageData =
+       formatLanguageSpecificData(eventData.languageSpecificData);
+     const message = createMessage(
+       event,
+       formattedLanguageData.description,
+       formattedLanguageData.title,
+       formattedLanguageData.eventPlace,
+       formattedLanguageData.languageKeys
+     );
+     if (event.status === "active" && !event.telegramMessageId) {
```

Comment:
this condition duplicates with line 9, telegramMessageId will always be falsy here